fma_project_spec.ipynb

```diff
@@ -0,0 +1,224 @@
+    "\n",
+    "Not so useful: \"The student has a basic familiarity with `pandas`.\"\n",
+    "\n",
+    "More useful: \"The student knows how to read in a csv file using `pandas` and how to compute grouped summary statistics using `groupby()`.\"\n",
+    "\n",
+    "> The student knows how to read in csv and json files using `pandas`, and examine correlations across the columns of a `pandas` DataFrame. The student understands basic plotting (scatter plots, pair plots) with `seaborn`. The student is also familiar with implementing both principal component analysis and supervised learning methods (k-fold crossvalidation) in `scikit-learn`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2. Project narrative intro"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## These recommendations are so on point! How does this playlist know me so well?\n",
+    "\n",
+    "> Over the past few years, streaming services with vast catalogues have increasingly become the primary means through which most people listen to their favourite music. But at the same time, the wealth of options on offer can mean users might be a bit overwhelmed when trying to look for newer music that suits their tastes. \n",
+    "\n",
+    "> For this reason, streaming services have found various means of categorizing music to allow for personalized recommendations. One method involves direct analysis of the raw audio information in a given song, scoring the raw data on a variety of metrics. Today, we'll be examining data compiled by a research group known as The Echo Nest. Our goal is to look through this dataset and classify songs as being either 'Hip-Hop' or 'Rock' - all without listening to a single one ourselves."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Read in the data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> To begin with, let's load the data on our tracks alongside the track metrics compiled by The Echo Nest. These exist in two different files, which are in different formats - `csv` and `json`. While `csv` is a popular file format for denoting tabular data, `json` is another common file format in which databases often return the results a given query. \n",
+    "\n",
+    "> Let's start by creating two `pandas` DataFrames out of these files, and then merging the DataFrames into one for our genre classification analysis."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "tracks = pd.read_csv('./fma-rock-vs-hiphop.csv')\n",
+    "tracks.info()\n",
+    "track_metrics = pd.read_json('./echonest-clean.json', precise_float=True)\n",
```

Comment:
We should either mention or remove `precise_float`, I don't think they would know to use it here.